config/application.yml.default

```diff
@@ -430,6 +433,7 @@ test:
   piv_cac_service_url: https://localhost:8443/
   piv_cac_verify_token_secret: 3ac13bfa23e22adae321194c083e783faf89469f6f85dcc0802b27475c94b5c3891b5657bd87d0c1ad65de459166440512f2311018db90d57b15d8ab6660748f
   piv_cac_verify_token_url: https://localhost:8443/
+  proofing_lambda_http_callback: false
```

Comment:
This logs a warning in build:

```
WARNING: Use strings for Figaro configuration. false was converted to "false".
```

Seems to be pretty common mistake. Probably not cause for concern in most cases, but could be worth adding some automated validation for this file.

Reply:
I think we can remove this even since it doesn't appear to be used